app/actors/hyrax/actors/create_with_remote_files_actor.rb

```diff
@@ -18,12 +18,34 @@ def update(env)
 
       private
 
+        def whitelisted_ingest_dirs
```

Comment:
I feel like it would be beneficial to have the method name match the configuration or vice-versa.